src/go/k8s/pkg/resources/configmap.go

```diff
@@ -204,16 +204,16 @@ func (r *ConfigMapResource) createConfiguration(
 			tls,
 		}
 	}
-	if r.pandaCluster.Spec.Configuration.TLS.AdminAPI.Enabled {
-		name := internal
+	name := internal
```

Comment:
what about external

Reply:
that's handled here https://github.com/vectorizedio/redpanda/pull/1163/files#diff-74fd28b7b800bebf72d52c4b3beaab538ca2605ce41735a05fcd6b4fd9b3963fR211